p2p/src/behaviour.rs

```diff
@@ -112,7 +114,13 @@ where
             } else {
                 BitswapConfig::default_client_mode()
             };
-            Some(Bitswap::new(peer_id, BitswapStore(rpc_client), bs_config).await)
```

Comment:
Should we remove the `rpc_client` arg to this function and instead pass in the SQLiteBlockStore ?  This way the path to the sql db doesn't need to be known in multiple places. Just where we make the first connection pool?